_docs/payout-details.md

```diff
@@ -114,6 +114,7 @@ Access Bank: 044
 FCMB Bank: 214
 United Bank for Africa: 033
 Zenith International: 057
+Fidelity Bank: 070
```

Comment:
Little comment @arnaugiro ... can we have it in alphabetical order like for NGN?